### DISCUSSION TITLE
Programatically close expanded rows?

### DISCUSSION THREAD

**Discussion by jakobluthman** [HC200ok/vue3-easy-data-table#118] [Q&A] [answered]
Does such functionality exist on this component? I can't seem to find anything about it in the documentation.

If this is not implemented, I'd be happy to make a pull request with such functionality.

Cheers!

**Comment by HC200ok** [comment, marked as answer]
@jakobluthman Programmatically closing expanded rows is not an existing feature, PR is welcome.
